NewUser'sGuide.md

```diff
@@ -0,0 +1,250 @@
+# New User’s Guide to Lutris
+
+Welcome to Lutris, [a game library manager](https://github.com/lutris/lutris) that allows you to bring your entire game collection together regardless of platform.
+
+This is a guide for new users that are just starting out with Lutris. Once you have [installed the Lutris client](https://lutris.net/downloads) for your distribution, [updated your Nvidia and AMD/Intel drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md) and have a [recent version of WINE](https://wiki.winehq.org/Download), you’ll be ready to go on your Lutris journey.
```

Comment:
We don't require a Wine version installed, it just so happens that installing Wine is the easiest way to pull in all dependencies. And it should link to https://github.com/lutris/docs/blob/master/WineDependencies.md.